### PR TITLE
Add kwargs to InferencePipeline run() method

### PR DESCRIPTION
**Pull request by mialsy**
Add kwargs support for InferencePipeline's` run()` method.

### CODE REVIEW

**Comment by mialsy** [comment]
May I get a review from @markblee or @ruomingp on this PR? Thanks in advance!

**Review by markblee — APPROVED** [review]

**Comment by mialsy** [comment]
Thanks for reviewing @markblee.  May I request write access for this repo to merge this PR?

**Pull request merged**
